shell/platform/tizen/tizen_window_elementary.cc

```diff
@@ -173,7 +173,7 @@ void TizenWindowElementary::RegisterEventHandlers() {
       [](void* data, Evas* evas, Evas_Object* object, void* event_info) {
         auto* self = reinterpret_cast<TizenWindowElementary*>(data);
         if (self->view_delegate_) {
-          if (self->elm_win_ == object) {
+          if (self->image_ == object) {
```

Comment:
Even when evas_object_event_callback_del , the target must be an image
Please check UnRegisterEventHandlers method.